datm/atm_comp_nuopc.F90

```diff
@@ -448,7 +448,7 @@ subroutine ModelAdvance(gcomp, rc)
     type(ESMF_Time)         :: nextTime
     type(ESMF_TimeInterval) :: timeStep
     real(r8)                :: nextsw_cday
-    logical                 :: restart_write ! restart alarm is ringing
+    logical                 :: rstwr         ! restart alarm is ringing
```

Comment:
Why change the name to a shorter name? What was wrong with restart_write?

Reply:
nothing, it was a cut and paste change. I've updated to restart_write.